src/main/java/org/sagebionetworks/bridge/spring/controllers/Exporter3Controller.java

```diff
@@ -35,4 +39,13 @@ public Exporter3Configuration initExporter3() throws BridgeSynapseException, IOE
         UserSession session = getAuthenticatedSession(DEVELOPER);
         return exporter3Service.initExporter3(session.getAppId());
     }
+
+    @PostMapping("/v5/studies/{studyId}/exporter3")
+    @ResponseStatus(HttpStatus.CREATED)
+    public Exporter3Configuration initExporter3ForStudy(@PathVariable String studyId) throws BridgeSynapseException, IOException,
+            SynapseException {
+        UserSession session = getAuthenticatedSession(STUDY_DESIGNER, DEVELOPER);
```

Comment:
Just noting the roles here don't match the SDK. Looking at it now, I think this is correct and the SDK should just change to reflect these.